src/Pages/ResList/StoreList.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { useNavigate, useSearchParams } from 'react-router-dom';
+import MenuItem from './MenuItem';
+import './StoreList.scss';
+
+const StoreList = () => {
+  const navigate = useNavigate();
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const menuListClick = e => {
+    searchParams.set('address', e.target.childNodes[0].data);
+    searchParams.set('menu', e.target.childNodes[2].data);
+    searchParams.set('category', '');
+    searchParams.set('limit', e.target.childNodes[4].data);
+    setSearchParams(searchParams);
+    navigate(`/resultlist?${searchParams.toString()}`);
+  };
+
+  return (
+    <div className="storeList">
+      <div className="storeListContainer">
+        <div className="storeListTitle">
+          <span>멜론 플레이트 추천 맛집 ! </span>
+          <span>인기 식당을 한눈에 확인하세요</span>
+        </div>
+
+        <div className="menuItemList">
+          {STORE_SORT.map(e => (
+            <MenuItem
+              key={e.img}
+              address={e.address}
+              menu={e.menu}
+              limit={e.limit}
+              img={e.img}
+              menuListClick={menuListClick}
+            />
+          ))}
```

Comment:
```suggestion
          {STORE_SORT.map(({img, address, menu, limit, img}) => (
            <MenuItem
              key={img}
              address={address}
              menu={menu}
              limit={limit}
              img={img}
              menuListClick={menuListClick}
            />
          ))}
```

e 라는 이름은 보통 event의 약자로 많이 사용 되기 때문에 혼동을 줄 수 있습니다.
매개변수의 이름도 특정할 수 있는 이름으로 지어주는 것이 좋고,
여러개의 프로퍼티를 사용해야 한다면 구조분해 할당을 통해 구현해 주는 것이 추후 유지보수에 좋습니다!